src/audio/taAudio.h

```diff
@@ -178,8 +178,6 @@ class TAAudio {
     virtual std::vector<String> listAudioDevices();
     bool initMidi(bool jack);
     void quitMidi();
-    /** remove and reload midi to allow hotswaping midi devices */
```

Comment:
Reinitializing RT midi is not required for hot swapping. Reference: https://www.music.mcgill.ca/~gary/rtmidi/#start

Reply:
Great to know. I was worried that wasn't possible.